studio/pages/project/[ref]/settings/logs/[type].tsx

```diff
@@ -256,18 +255,16 @@ export const LogPage: NextPage = () => {
               />
             </div>
             <div className="flex flex-row justify-end p-2 w-full">
-              <Flag name="logsCustomSql">
-                {isSelectQuery && (
-                  <div className="flex flex-grow flex-row items-center gap-x-1">
-                    {/* // we don't have a slim Alert component yet */}
-                    <IconInfo size="tiny" />
-                    <Typography.Text small={true} type="secondary">
-                      Custom queries are restricted to a {type === 'database' ? '2 hour' : '7 day'}{' '}
-                      querying window.
-                    </Typography.Text>
-                  </div>
-                )}
-              </Flag>
+              {isSelectQuery && (
+                <div className="flex flex-grow flex-row items-center gap-x-1">
+                  {/* // we don't have a slim Alert component yet */}
```

Comment:
You can use the <InformationBox /> component for this? just pass the `title` prop

Reply:
done, updated the InformationBox component as well